### DISCUSSION TITLE
Backoffice operations add-edit-delete on Customer / Addresses are not logged in "Logs"

### DISCUSSION THREAD

**Discussion by gennaris** [PrestaShop/PrestaShop#35507] [Ideas]
#### Describe the bug

Backoffice operations add-edit-delete on Customer / Addresses are not tracked in "Logs" 

#### Expected behavior

I expect BO operations on all core objects from employees being logged , this is crucial for an admin to track employee activities. This is already correctly happening for products.

I see this open discussion already => https://github.com/PrestaShop/PrestaShop/issues/24698
but the actual PS 1.7 "logging" situation is unclear to me.

#### Steps to Reproduce

1 Add / edit or delete a product
2. Go in Advanced Parameters > Logs
3. See the operation is being tracked correctly with an informative log :
 
![2021-10-21_11-48-37](https://user-images.githubusercontent.com/43071148/138253969-b3f3a7ca-08be-4754-9d62-4a7cbbcfc2cd.png)

4. Add / edit or delete a customer
5. Go in Advanced Parameters > Logs
6. Be sad since nothing is tracked :(

#### Additional information

* PrestaShop version: 1.7.8.0
* PHP version: 7.3

**Comment by hibatallahAouadni** [comment]
Hello @gennaris 

I was able to reproduce the issue with PrestaShop version 1.7.7.8. I’ll add this to the debug backlog so that it’s fixed.

Please be aware that due to the high number of reported issues, some bugs might take a very long time to be resolved. If this issue is important to you and you cannot wait for it to be fixed on the project’s own time, we strongly suggest you consider [hiring a specialist](https://www.prestashop.com/en/experts) to help you. 

If you manage to get the problem fixed, then please contribute the fix back to the project as a [Pull Request](https://devdocs.prestashop.com/1.7/contribute/contribute-pull-requests/). Remember that the more people contribute, the better PrestaShop becomes for everyone.

Thank you

**Comment by eopws** [comment]
Please review #26366

**Comment by hibatallahAouadni** [comment]
@eopws thanks for your PR :rocket: 
Our developer team have already reviewed your PR, so please check their comments :wink:

**Comment by HanaRebaiQA** [comment]
Hello,

This issue is partially treated. Only the Customer part is done with this available PR [#26366](https://github.com/PrestaShop/PrestaShop/pull/26366)
Customers part : not fixed
Adresses part : PR is needed.
https://watch.screencastify.com/v/z7FTbPoETuRnYkxYxFeE

Thanks!

**Comment by RobMeerwijk** [comment]
Starting a few weeks ago we experience the same bug. Lots of warning emails and lots of loglines showing:

Log ID (65250f34a0b4a8.72529903) | Failed to find customer address . context: [] 

We have 8.1.2 installed, but it happened also in 8.1.1

Serverinformatie: Linux #1 SMP PREEMPT_DYNAMIC Debian 6.1.55-1 (2023-09-29) x86_64
Server softwareversie: Apache/2.4.57 (Debian)
PHP-versie: 8.1.23
Geheugenlimiet: 512M
Maximale uitvoeringstijd: 30
Maximale bestandsgrootte voor uploads: 1024M
MySQL versie: 10.3.37-MariaDB-log 

Related to [#33813]

**Reply by NickBouwhuis** [reply]
Did you ever find out what was causing it? It started for me a few days ago.

**Comment by EmyM668** [comment]
Hello, 
have you found a solution. Personally this problem has just appeared since I installed a new payment module (Mollie). I use Paypal and the Mollie payment module. Error Log ID (XXabf2df8f28b9.XXXXX) | Failed to find customer address. context:[]
 only appears when creating an order and customer account with Mollie. No errors with Paypal. No errors either when creating a customer account without an order.
The error appears whether the order account is in customer status activated account or Guest.
I contacted the module supplier who for the moment refers me to Github, considering that PrestaShop is the problem.
Prestashop 1.7.8.2

**Reply by DanielvdAkker** [reply]
Hi Emy,
I have the same problem. Tried to deinstall and re-install Mollie, then there is no problem. However, all Mollie-settings are gone, so customers are not able to make payments again... The logging is gone, but when you add the Mollie-settings, the log-problems are back again... So it has to be in Mollie somewhere...

**Comment by Juiliard** [comment]
the same on PS 8.2 and Mollie 6.2.3
